packages/serverpod_serialization/lib/src/extensions/serialization_extensions.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:typed_data';
+
+import 'package:serverpod_serialization/serverpod_serialization.dart';
+
+/// Expose toJson on DateTime
+/// Expose static fromJson builder
+extension DateTimeExt on DateTime {
```

Comment:
Suggestion: Rename to `DateTimeJsonExtention`.

Since the extensions are only connected to our `fromJson` and `toJson` we might as well be specific.

Reply:
Make the same changes for the other extensions.